sesam.py

```diff
@@ -1227,6 +1262,12 @@ def download(self):
             zip_data = self.format_zip_config(zip_data)
             zip_config = zipfile.ZipFile(io.BytesIO(zip_data))
             zip_config.extractall()
+            if not self.args.is_connector:
```

Comment:
Suggestion: move if "self.args.jinja_vars" in replace_template_variables() out to where it's being called, it makes it easier to see that it's skipped if no such arg is present